nl.emphisia.icon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "nl.emphisia.icon",
```

Comment:
Can you add your GitHub account to https://www.emphisia.nl/ ?

Also will you be able to deploy a page at emphisia.nl/.well-known ? The site goes to `www.emphisia.nl`

Reply:
@bbhtt Yes, that is no issue. I have added my github to the site!

Reply:
@bbhtt That is no issue at all! I have added my github to my website :)